benchmarks/src/generated/main/grpc/io/grpc/benchmarks/proto/BenchmarkServiceGrpc.java

```diff
@@ -268,11 +280,21 @@ public void invoke(Req request, io.grpc.stub.StreamObserver<Resp> responseObserv
     }
   }
 
-  private static final class BenchmarkServiceDescriptorSupplier implements io.grpc.protobuf.ProtoFileDescriptorSupplier {
+  private static final class BenchmarkServiceDescriptorSupplier implements io.grpc.protobuf.ProtoDescriptorSupplier {
     @java.lang.Override
     public com.google.protobuf.Descriptors.FileDescriptor getFileDescriptor() {
       return io.grpc.benchmarks.proto.Services.getDescriptor();
```

Comment:
> The main reason I've done this is because in order to get service/method descriptors you need to pass index

How so? Using the `find*` methods doesn't. I thought my suggested code above would work fine without any indexes.

Even if the `find*` methods weren't available, we could write them ourselves in a utility by simply looping over the services/methods until we find the right one.